src/modules/routes/RouteData.ts

```diff
@@ -1438,6 +1438,11 @@ Routes.add(new RegionRoute(
     'Unova Route 12', Region.unova, 12,
     new RoutePokemon({
         land: ['Pinsir', 'Heracross', 'Roselia', 'Combee', 'Vespiquen', 'Tranquill', 'Sewaddle'],
+        special:
+        [
+            new SpecialRoutePokemon(['Cherrim (Overcast)'], new ObtainedPokemonRequirement('Cherrim (Overcast)')),
+	    new SpecialRoutePokemon(['Cherrim (Sunshine)'], new MultiRequirement([new ObtainedPokemonRequirement('Cherrim (Sunshine)'), new WeatherRequirement([WeatherType.Harsh_Sunlight])])),
```

Comment:
```suggestion
            new SpecialRoutePokemon(['Cherrim (Overcast)'], new ObtainedPokemonRequirement('Cherrim (Overcast)')),
	        new SpecialRoutePokemon(['Cherrim (Sunshine)'], new MultiRequirement([new ObtainedPokemonRequirement('Cherrim (Sunshine)'), new WeatherRequirement([WeatherType.Harsh_Sunlight])])),
```
You should indent the second line here.
Also at first glance it seems weird to have Overcast show up alongside Sunshine, but I don't know if you have a reason for that or because you forgot to include weather requirements for Overcast.